File: php/Mahasiswa.php
<?php
class Mahasiswa extends Siak{
    private $nim, $prodi, $fakultas;

    public function __construct($nim = 0, $prodi = "", $fakultas = ""){
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    public function setNim($nim){
        $this->nim = $nim;
    }
    public function getNim(){
        return $this->nim;
    }

    public function setProdi($prodi){
        $this->prodi = $prodi;
    }
    public function getProdi(){
        return $this->prodi;
    }

    public function setFakultas($fakultas){
        $this->fakultas = $fakultas;
    }
    public function getFakultas(){
        return $this->fakultas;
    }

    public function __destruct(){

    }
}
?>

File: python/Mahasiswa.py
from Siak import Siak

class Mahasiswa(Siak):
    
    def __init__(self):
        self.nim = 0
        self.fakultas = ""
        self.prodi = ""
        
    def setNim(self, nim):
        self.nim = nim
    
    def setFakultas(self, fakultas):
        self.fakultas = fakultas
        
    def setProdi(self, prodi):
        self.prodi = prodi
    
    
    def getNim(self):
        return self.nim
    
    def getFakultas(self):
        return self.fakultas
    
    def getProdi(self):
        return self.prodi